src/main.rs

```diff
@@ -1,81 +1,80 @@
-use getopts::Options;
-use std::{collections::HashMap, error::Error, io::Write, num::ParseIntError, ops::AddAssign};
+use std::{collections::HashMap, error::Error, fs, io::Write, num::ParseIntError, ops::AddAssign};
 
 fn main() -> Result<(), Box<dyn Error>> {
     let mut opts = getopts::Options::new();
     let opts = opts
-        .optflag("q", "quiet", "Hides the output of commit messages.")
+        .optflag("q", "quiet", "Hide the output of commit messages.")
         .optflag("h", "help", "Show this help and exit.")
         .optmulti(
             "a",
             "author-contains",
-            "Filters after certain author names. ORs if multiple specified.",
+            "Filter for certain author names. ORs if multiple specified.",
```

Comment:
"Filter after" seemed to be "not the yellow from the egg"; it means here the temporal order of something.  Since it was obviously intended to describe the extraction of information by a certain filter criterion, I changed it to "filter for".

Reply:
Yeah, prepositions are my Achilles heal 😅

Reply:
Just for completeness:  "heel" (part of the leg), not "heal" (to cure).